controller/controller.wasm.go

```diff
@@ -0,0 +1,102 @@
+//go:build wasm
+
+package contoller
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"reflect"
+
+	"github.com/HexmosTech/gabs/v2"
+	"github.com/HexmosTech/httpie-go"
+	"github.com/HexmosTech/lama2/cmdexec"
+	"github.com/HexmosTech/lama2/cmdgen"
+
+	"github.com/HexmosTech/lama2/parser"
+	"github.com/HexmosTech/lama2/preprocess"
+
+	"syscall/js"
+)
+
+func GetParsedAPIBlocks(parsedAPI *gabs.Container) []*gabs.Container {
+	return parsedAPI.S("value").Data().(*gabs.Container).Children()
+}
+
+func ExecuteProcessorBlock(block *gabs.Container) {
+	b := block.S("value").Data().(*gabs.Container)
+	script := b.Data().(string)
+	js.Global().Call("eval", script)
+}
+
+func ExecuteRequestorBlock(block *gabs.Container) httpie.ExResponse {
+	preprocess.ProcessVarsInBlock(block)
+	// TODO - replace stuff in headers, and varjson and json as well
+	cmd, stdinBody := cmdgen.ConstructCommand(block)
+	resp, e1 := cmdexec.ExecCommand(cmd, stdinBody)
+	printFields(resp)
```

Comment:
What does this do

Reply:
Added it for debugging purpose, printing the response